addons/sale_timesheet/views/product_views.xml

```diff
@@ -6,8 +6,18 @@
         <field name="inherit_id" ref="sale.product_template_form_view"/>
         <field name="arch" type="xml">
             <field name="product_tooltip" position="after">
-                <label for="product_tooltip" string="" invisible="type != 'service' or service_policy != 'ordered_prepaid' or not sale_ok"/>
-                <div invisible="type != 'service' or service_policy != 'ordered_prepaid' or not sale_ok" class="fst-italic text-muted">
+                <label for="product_tooltip" string=""
```

Comment:
Seems strange to me, shouldn't it be the label for `service_upsell_threshold` instead ?
(or can't we use the `nolabel` attribute ?)

Reply:
Yes I guess :thinking: (I think they used product_tooltip because they are adding tooltip which contains `service_upsell_threshold`)
(`nolabel` will remove that label space so string will not be aligned to values that's why they are using `string=""` so empty space for label stay as it is)